UnityProject/Assets/Scripts/Items/Implants/Limbs/HumanoidLeg.cs

```diff
@@ -0,0 +1,51 @@
+using Player.Movement;
+using UnityEngine;
+
+namespace HealthV2.Limbs
+{
+	public class HumanoidLeg : Limb, IMovementEffect
```

Comment:
maybe just call this ``Leg`` since the other limb is simply ``Arm``

Reply:
It makes sense to call it "Humanoid[limb]" because there can be many types of legs in the future.

Reply:
renamed arm to HumanoidArm